### ISSUE TITLE
is there any guide for translation?

### ISSUE THREAD

**Issue by benjioh5** [crypto101/book#362]
I want to translate this book to korean. But I could not find translation guide and tex build script for multi language support. Can I make it? Or just fork it for korean translation?

**Comment by multun** [comment]
@benjioh5 things aren't perfect yet but you can try out #372

**Comment by benjioh5** [comment]
@multun Thanks, I'm in

**Comment by multun** [comment]
@benjioh5 are you still interested in translating some of the content? If time and motivation are a concern, I'd suggest finding translation buddies to help out (I do the same for the french translation, and it has worked pretty well so far)

**Comment by benjioh5** [comment]
@multun Yes, but I have really no time for this translation. Is there any korean translater on this project?

I think i can start translation after 12/25/2020

**Comment by multun** [comment]
@benjioh5 you're the only one to have shown interest so far (which is pretty cool already, many wouldn't want to)

**Comment by benjioh5** [comment]
@multun I'll call my buddies who can translate. Just wait more pleaze. Thanks

**Comment by benjioh5** [comment]
@multun Could you check [transifex](https://www.transifex.com/crypto101/crypto101/)? I sent permission request. I'll invets 5~6 hours per week for translation.

**Comment by multun** [comment]
oh wow that's awesome! I hope you didn't feel pressured to commit that much time, I was just curious, really. This book didn't have any translation for quite a while and I'm pretty sure we can all live with a few months more delay ;-)
